components/feature/top-sites/src/main/res/values/mozac_feature_top_sites_strings.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Default title for pinned Baidu top site that links to Baidu home page  -->
+    <string name="default_top_site_baidu" translatable="false">百度</string>
+    <!-- Default title for pinned JD top site that links to JD home page  -->
+    <string name="default_top_site_jd" translatable="false">京东</string>
```

Comment:
I am a bit suspicious about this and perhaps we should just move these strings to strings.xml. In my manual testing on Fenix where I set my region to CN, I somehow get the english names JD and Baidu, but I get the Chinese naming when my device language is set to Chinese. I am wondering if this is because of the local string translation that is already in Fenix that also uses the same exact string name.